src/Conf.ml

```diff
@@ -514,6 +515,23 @@ module Formatting = struct
     C.choice ~names ~all ~env ~doc ~allow_inline:true ~update:(fun conf x ->
         {conf with sequence_style= x} )
 
+  let structure_item_grouping =
+    let doc = "Line breaks between structure items." in
+    let env = Arg.env_var "OCAMLFORMAT_STRUCTURE_ITEM_GROUPING" in
+    let names = ["structure-item-grouping"] in
+    let all =
+      [ ( "sparse"
+        , `Sparse
+        , "$(b,sparse) will always break a line between two structure items."
```

Comment:
maybe reword to match other case: 'will always leave an open line between two items of a structure or signature`